spec/controllers/audience_controller_spec.rb

```diff
@@ -32,17 +32,20 @@
     end
   end
 
-  describe "GET export" do
-    it_behaves_like "authorize called for action", :get, :export do
+  describe "POST export" do
```

Comment:
could we add a spec to ensure the email is sent to the imperonsating_user during impersonations?